media-tv/w_scan/w_scan-20170107.ebuild

```diff
@@ -13,12 +13,13 @@ KEYWORDS="~amd64 ~x86"
 IUSE="doc examples"
 
 DEPEND=">=virtual/linuxtv-dvb-headers-5.8"
-RDEPEND=""
 
-src_install() {
-	emake DESTDIR="${D}" install
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
+DOCS=( ChangeLog README )
```

Comment:
check whether the catch all of `einstalldocs` catches too much, and if not, omit it